samples/Chat/package.json

```diff
@@ -18,7 +19,9 @@
     "webpack": "rushx _by-flavor \"webpack --mode=production --env production\"",
     "_current-flavor": "echo You are running under COMMUNICATION_REACT_FLAVOR: && env-cmd -f ../../common/config/env/.env node -p process.env.COMMUNICATION_REACT_FLAVOR",
     "_by-flavor": "rushx _current-flavor && env-cmd -f ../../common/config/env/.env --use-shell",
-    "_typecheck": "rushx _by-flavor \"if-env COMMUNICATION_REACT_FLAVOR=stable && echo skip || (if-env COMMUNICATION_REACT_FLAVOR=beta && tsc)\""
+    "_stable-flavor": "rushx _current-flavor && env-cmd -f ../../common/config/env/stable.env --use-shell",
+    "_typecheck": "rushx _by-flavor \"if-env COMMUNICATION_REACT_FLAVOR=stable && echo skip || (if-env COMMUNICATION_REACT_FLAVOR=beta && tsc)\"",
+    "_preprocess:stable-flavor": "rushx _stable-flavor \"rushx preprocess && rushx lint:fix && rushx prettier\""
```

Comment:
These `package.json` files are getting out of hand with the number of scripts. The problem is:

* Helper scripts like "_typecheck" create noise
* There is no documentation, help message -- nothing ergonomic.

I'd like us to start adding `scripts/*mjs` instead that are invoked from `package.json`. See https://github.com/Azure/communication-ui-library/blob/main/packages/react-composites/scripts/runBrowserTests.mjs for example.

I'd recommend merging this PR as is, and then iterating on top to extract some helper scripts with better help / documentation.

Reply:
I'm not quite sure what is happening in the file you are linking to. Could we set up a meeting about this either tuesday or when I get back on the 3rd? is this a script file that depending on the flags you give as args runs different command line functions like you would see in package.json?